code/modules/gear_presets/contractor.dm

```diff
@@ -4,7 +4,7 @@
 	rank = JOB_CONTRACTOR
 	idtype = /obj/item/card/id/data
 	faction = FACTION_CONTRACTOR
-	faction_group = FACTION_LIST_ERT
+	faction_group = FACTION_CONTRACTOR
```

Comment:
faction_group is usually a list, use the _list one (or make up a new one if you want them to be groupped with other factions)